bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/device/GroupMessageStateMachine.java

```diff
@@ -132,7 +135,13 @@ public boolean action(GroupMessage a, InsteonAddress address, int group) {
             case EXPECT_CLEAN:
                 switch (a) {
                     case BCAST:
-                        publish = false;
+                        if (lastCmd1 == cmd1) {
+                            publish = false;
+                        } else {
+                            logger.debug("{} group {} cmd1 {} is not a dup BCAST, last cmd1 {}", address, group,
+                                    Utils.getHexByte(cmd1), Utils.getHexByte(lastCmd1));
```

Comment:
To prevent unnecessary calls to `Utils.getHexByte` we should check if logging would occur at all.
```suggestion
                            if (logger.isDebugEnabled()) {
                                logger.debug("{} group {} cmd1 {} is not a dup BCAST, last cmd1 {}", address, group,
                                         Utils.getHexByte(cmd1), Utils.getHexByte(lastCmd1));
                            }
```

Reply:
Good idea, I wonder why this convention isn't used around all logger.* message.